.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: ruby
+rvm:
+  - 2.0
+  - 2.1
```

Comment:
Be sure to use strings for these.  Travis treats "2.0" to mean "the latest of Ruby 2.0" whereas using a float value does not do that.